README.md

```diff
@@ -28,10 +41,17 @@ af.init(appId, devKey, function(err, events) {
     af.sendTrackingWithEvent(eventName, {key1: value1, key2: value2}, function(err, events) {
         //events is {key1: value1, key2, value2}
     });
+})
+```
 
-    //To get appsFlyerUID an event,
-    af.getAppsFlyerUID(function(err, uid) {
```

Comment:
The af.getAppsFlyerUID does work in the IOS version, so can you maintain the method call under the IOS example. Unless you're able to add support for the af.getAppsFlyerUID method to Android?